src/main/java/com/ecore/roles/service/TeamsService.java

```diff
@@ -0,0 +1,13 @@
+package com.ecore.roles.service;
+
+import com.ecore.roles.client.model.Team;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface TeamsService {
+
+    Team getTeam(UUID id);
```

Comment:
It could be just called "get" or "getById", because it's inside of the teams service.